packages/keyring-snap-client/src/KeyringSnapRpcClient.test.ts

```diff
@@ -1,6 +1,6 @@
+import type { KeyringAccount } from '@metamask/keyring-api';
```

Comment:
The `KeyringAccount` lives on the `keyring-api`, the `InternalAccount` on `keyring-internal-api`.